src/request/repository/converter/request-converter.js

```diff
@@ -139,19 +140,25 @@ var getStatusCodeText = (function() {
 })();
 var setupIndex = function(request, type, payload) {
     if (type == 'begin-request') {
-        request._requestStartTime = payload.requestStartTime;
-        request._requestMethod = payload.requestMethod;
-        request._requestUrl = (payload.requestPath || '') + (payload.requestQueryString || '');
+        var url = parse(payload.url);
+        payload.path = url.pathname;
+        payload.query = url.query;
+        
+        request._requestStartTime = payload.startTime;
+        request._requestMethod = payload.method;
+        request._requestUrl = payload.path + payload.query;
         request._requestIsAjax = payload.requestIsAjax;
     }
     if (type == 'end-request') {
-        request._responseStatusCode = payload.responseStatusCode; 
-        request._responseStatusText = getStatusCodeText(payload.responseStatusCode);
-        request._responseContentType = payload.responseContentType;
-        request._responseContentCategory = getContentTypeCategory(payload.responseContentType);  
+        var contentType = payload.headers && payload.headers['Content-Type'];
```

Comment:
I'm not necessarily opposed to promoting `Content-Type` to avoid this issue, but it's a bit of a slippery slope. `Content-Length` could be next, followed by who knows what, as we keep bumping into use cases where we want to do more than just display a value.

All that said, I agree with @mike-kaufman here. Let's just solve the problem once and for all by normalizing the keys. (But don't throw away the original casing since we'll want to display it that way!)

Apparently [lodash makes this easy to do](https://lodash.com/docs#transform) (is that already a dependency?): 

```javascript
var normalizedHeaders = _.transform(headers, function (result, value, key) {
    result[key.toLowerCase()] = value;
});
```